.github/workflows/test.yml

```diff
@@ -0,0 +1,310 @@
+name: SalesforceCommerceCloud/pwa-kit/test
+on:
+  schedule:
+    - cron: 0 8 * * *
+  push:
+     branches:
+     # TODO: Before merging remove branches filter
+     - github-actions
+jobs:
+  pwa-kit:
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [14]
+        npm: [6, 7, 8]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: npm
+
+      - name: Save NPM version
+        run: |-
+            echo "NPM_VERSION=${{ matrix.npm }}" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Setup Ubuntu Machine
+        uses: "./.github/actions/setup_ubuntu"
+
+      - name: Install node dependencies with NPM version
+        if: matrix.npm != 6
+        run: |- 
+          npm install -g npm@${{ matrix.npm }}
+          npm ci
```

Comment:
do we still need this if we already have the npm matrix? Also, the `npm ci` defeat the purpose of the cache from the `actions/setup-node@v3` action. 

I think `actions/setup-node@v3` should be sufficient and we can just remove this step?